src/components/SQFormScrollableCard/SQFormScrollableCard.js

```diff
@@ -240,7 +240,7 @@ SQFormScrollableCard.propTypes = {
    * */
   validationSchema: PropTypes.object,
   //** Boolean used to determine if title/header is enabled or disabled */
-  isHeaderDisabled: PropTypes.bool
+  isHeaderDisabled: PropTypes.bool,
```

Comment:
I think this can be removed form propTypes.

Reply:
It looks like the new MenuWrapper component uses the isHeaderDisabled Prop, I think ill hold off on making this change until i look at that new component a bit closer, just incase!